sentry-android-core/src/main/java/io/sentry/android/core/ActivityLifecycleIntegration.java

```diff
@@ -176,7 +175,9 @@ private void stopPreviousTransactions() {
 
   private void startTracing(final @NotNull Activity activity) {
     WeakReference<Activity> weakActivity = new WeakReference<>(activity);
-    if (performanceEnabled && !isRunningTransaction(activity) && hub != null) {
+    if (!performanceEnabled && hub != null) {
```

Comment:
hm, should we check if there's a trace started from this activity still running? similar to `isRunningTransaction`

Reply:
You mean so we don't start too many? Yeah we can.

Reply:
Just pushed some changes, can you please give the last commit another pass @romtsn